src/mscorlib/mscorlib.shared.sources.props

```diff
@@ -732,7 +732,6 @@
     <ThreadingSources Include="$(BclSourcesRoot)\System\Threading\Timer.cs" />
     <ThreadingSources Include="$(BclSourcesRoot)\System\Threading\Volatile.cs" />
     <ThreadingSources Include="$(BclSourcesRoot)\System\Threading\WaitHandle.cs" />
-    <ThreadingSources Include="$(BclSourcesRoot)\System\Threading\WaitHandleExtensions.cs" />
```

Comment:
Could you please just put a FeatureCoreClr condition on the file, without removing it? It will break full framework build otherwise. We can still remove the file (there is actually a bunch of them with the condition) ... but it should done from the TFS side.